fi.json

```diff
@@ -82,301 +82,301 @@
-            "msg-font-found": "This font is detected on your system.",
-            "label-font-name": "Font name",
-            "option-community-themes": "Open community themes",
-            "option-community-themes-description": "Preview and use amazing themes created by the community.",
-            "button-browse-community-themes": "Browse",
-            "option-font-size": "Font size",
-            "option-font-size-description": "Font size in pixels that affects editor and preview.",
-            "option-font-size-action": "Quick font size adjustment",
-            "option-font-size-action-description": "Adjust the font size using Ctrl + Scroll, or using the trackpad pinch-zoom gesture.",
-            "option-themes": "Themes",
-            "option-manage-themes-description": "Manage installed themes and browse community themes.",
-            "option-theme-button-manage": "Manage",
-            "option-choice-none": "None",
-            "option-native-menus": "Native menus",
-            "option-native-menus-desc": "Menus throughout the app will match the operating system. They will not be affected by your theme.",
-            "button-reload-themes": "Reload themes",
-            "msg-reloaded-themes": "Reloaded custom CSS themes.",
-            "button-open-themes-folder": "Open themes folder",
-            "option-css-snippets": "CSS snippets",
-            "label-no-css-snippets-found": "No CSS snippets found in the snippets folder.",
-            "no-snippet-description": "CSS Snippets are stored in \"{{path}}\".",
-            "button-reload-snippets": "Reload snippets",
-            "button-open-snippets-folder": "Open snippets folder",
-            "msg-reloaded-snippets": "Reloaded CSS snippets.",
-            "option-toggle-snippet-description": "Apply CSS snippet at \"{{path}}\".",
-            "label-installed-themes": "Installed themes",
-            "label-screenshot-unavailable": "Screenshot unavailable",
-            "label-default-theme": "Default",
-            "tooltip-click-to-enlarge": "Click to enlarge",
-            "button-update": "Update",
-            "label-currently-selected": "Selected",
-            "option-frame-style": "Window frame style",
-            "option-frame-description": "Determines the styling of the title bar of Obsidian windows. Requires a full restart to take effect.",
-            "option-frame-hidden": "Hidden (default)",
-            "option-frame-obsidian": "Obsidian frame",
-            "option-frame-native": "Native frame",
-            "option-configure-ribbon": "Ribbon menu",
-            "option-configure-ribbon-desc": "Configure what commands appear in the ribbon menu in that navigation bar.",
+            "name": "Ulkonäkö",
+            "option-base-theme": "Pohjateema",
+            "option-base-theme-description": "Valitse Obsidianin oletusväriteema.",
+            "option-accent-color": "Aksenttiväri",
```

Comment:
Korostusväri

Reply:
Hello, thanks for the suggestion! Feel free to submit a PR to the source repo :)